code/modules/surgery/organs/heart.dm

```diff
@@ -231,3 +231,229 @@
 		owner.heal_overall_damage(15, 15, 0, BODYTYPE_ORGANIC)
 		if(owner.reagents.get_reagent_amount(/datum/reagent/medicine/ephedrine) < 20)
 			owner.reagents.add_reagent(/datum/reagent/medicine/ephedrine, 10)
+
+
+//NSV13 - Rest of file is purely ours from down here - Ethereal Crystalization
+
```

Comment:
Code looks alright. I would prefer this section in a separate file in the NSV folder.